functions/common/config-common/src/main/resources/META-INF/spring.factories

```diff
@@ -1,3 +1,2 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
-  org.springframework.cloud.fn.common.config.SpelExpressionConverterConfiguration,\
-  org.springframework.cloud.fn.common.config.ComponentCustomizationAutoConfiguration
+  org.springframework.cloud.fn.common.config.SpelExpressionConverterConfiguration
```

Comment:
Is it no longer auto-configured? How do we provide it to individual function/app modules?

Reply:
Manually. See my change to the `FileSupplierConfiguration`.
Also see my commit comment for more reasoning.

Reply:
Ah ok. That makes sense.